client/src/components/auth/ForgotPassword.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <div>
+        <h4 class="title-of-card pb-4">Reset your password</h4>
+        <p class="info-text-p">{{ lang.ENTER_YOUR_EMAIL_ADDRESS }}</p>
+        <VForm>
+            <VCol cols="11" sm="11" md="8" class="pa-0 py-4">
+                <label for="email">{{ lang.EMAIL }}*</label>
+                <VTextField
+                    id="email"
+                    v-bind:value="email"
+                    :error-messages="emailErrors"
+                    @blur="setEmail($event.target.value)"
```

Comment:
isn't better to use @input event? In this case the email will change each time you press key:

@input="setEmail"
```
setEmail(e) {
    this.email = e.target.value;
}
```